main/midi/midiPlayer.c

```diff
@@ -513,8 +513,8 @@ static int32_t midiSumOscillators(midiPlayer_t* player)
         playingVoices &= ~(1 << voiceIdx);
         midiVoice_t* voice = &(voices[voiceIdx]);
 
-        if (!(states[voiceIdx].on || states->held || states->sustenuto || states->attack || states->decay
-              || states->sustain || states->release)
+        if (!(states->on || states->held || states->sustenuto || states->attack || states->decay || states->sustain
```

Comment:
I thought I fixed this in `main` already but I guess it was only in some branch?

Reply:
Pretty sure it was just in the Big Bug branch. I made the same change locally but didn't include it here to avoid merge conflicts, since I thought Big Bug would get merged first.